apps/condo/domains/property/components/AddressSuggestionsSearchInput.tsx

```diff
@@ -55,13 +52,14 @@ export const AddressSuggestionsSearchInput: React.FC<AddressSearchInputProps> =
     const { addressApi } = useAddressApi()
 
     const searchAddress = useCallback(
-        async (query: string): Promise<TSelectItem[]> => {
+        async (query: string): Promise<TSelectedAddressSuggestion[]> => {
             try {
                 const { suggestions } = await addressApi.getSuggestions(query)
                 return suggestions.map(suggestion => {
                     return {
                         text: suggestion.value,
                         value: suggestion.rawValue,
+                        isHouse: validHouseTypes.includes(suggestion.data.house_type_full),
```

Comment:
isValidHouse

Reply:
It's not about validation. The customer may choose a city or street. This field shows that the customer has chosen a house.